src/templates/onboarding-connect/onboarding-connect.stories.js

```diff
@@ -0,0 +1,79 @@
+import { X } from 'lucide-react';
+import ProgressSteps from '../../components/progress-steps';
+import Topbar from '../../components/topbar';
+import Button from '../../components/button';
+import Label from '../../components/label';
```

Comment:
I noticed that you have imported each component on separate lines. To improve the readability of the code, you could consolidate these imports into a single line

```suggestion
import { ProgressSteps, Topbar , Button, Label, } from '@/components';
```